components/renku_data_services/project/db.py

```diff
@@ -163,6 +163,34 @@ async def get_project(self, user: base_models.APIUser, project_id: str) -> model
 
             return project_orm.dump()
 
+    async def get_project_by_namespace_slug(
+        self, user: base_models.APIUser, namespace: str, slug: str
+    ) -> models.Project:
+        """Get one project from the database."""
+        async with self.session_maker() as session:
+            stmt = (
+                select(schemas.ProjectORM)
+                .where(schemas.NamespaceORM.slug == namespace)
+                .where(schemas.ProjectSlug.namespace_id == schemas.NamespaceORM.id)
+                .where(schemas.ProjectSlug.slug == slug)
```

Comment:
Same as above: `slug.lower()`.

Reply:
Fixed in 651ceb960de0d5064cd9c283d7f01d0c1e56c02c